src/Frontend/Page/Comment.hs

```diff
@@ -50,7 +50,28 @@ instance ToHtml CommentWidget where
                         i_ [class_ "icon-flag"] nil
                         "melden"
         div_ [class_ "comment-replies"] $ do
-            for_ (comment ^. commentReplies) $ \reply -> do
-                div_ [class_ "comment-reply"] $ do
```

Comment:
> My hope was that the class comment-reply would be enough for styling.

good point.  if that's the case though, we can just revert to one widget type after styling.